### DISCUSSION TITLE
LegOverlay -1 on base strafing locomotion?

### DISCUSSION THREAD

**Discussion by darthbator** [Sixze/ALS-Refactored#96] [Q&A]
Hey! I was wondering why this is set?

<img width="842" alt="image" src="https://user-images.githubusercontent.com/2659942/176243260-ad63a11b-1f8e-45d9-b8f7-b722db3b5183.png">

It seems to always cause the legs to poke through when you're trying to overwrite them with an overlay. I was wondering what this is accomplishing in the base system? Looking through everything I can't see any reason why this is this way and it seems to work fine if it's removed. Is this an artifact from the original animations? They are setup this way as well but the original store package is not factored for leg overlays at all, and they are not really fully implemented in ALS community (the nodes are there but they are always set no to blend the overlay). 

BTW this plugin is a generally fantastic reorganization of ALS, it's been a joy to work with so far!

**Comment by Sixze** [comment]
This was done intentionally to remove the impact of the leg pose from overlay states on some complex movement animations. For example, if you remove this curve from the movement animations and look at the character's legs when strafing while aiming with a bow, you'll see that the legs have taken a somewhat unnatural pose: the character places them at a greater distance from each other than usual, and this looks strange.

The original ALS and ALS Community behave exactly the same.
